secant/Features/SendFlow/SendFlowStore.swift

```diff
@@ -225,12 +235,12 @@ extension SendFlowReducer {
 // MARK: - ViewStore
 
 extension SendFlowViewStore {
-    var bindingForTransaction: Binding<SendFlowTransaction> {
-        self.binding(
-            get: \.transaction,
-            send: SendFlowAction.updateTransaction
-        )
-    }
+//    var bindingForTransaction: Binding<SendFlowTransaction> {
```

Comment:
Should we just delete this commented out code?

Reply:
removed